website/addons/s3/requirements.txt

```diff
@@ -1 +1 @@
-boto==2.31.1
+boto==2.38.0
```

Comment:
The dependency is on the osf storage file audit scripts. Boto throws utf exceptions when uploading large files when it switches to multipart uploads in later versions. We can easily create a virtualenv to workaround this issue for the audit scripts. Don't let this be a blocker I'll correct this on the servers as needed.